index.css

```diff
@@ -0,0 +1,236 @@
+body {
+  margin: 0;
+}
+
+.font {
+  font-family: 'Poppins', sans-serif;
+}
+
+h1 {
+  color: #172b4d;
+  margin-top: 5px;
+  margin-bottom: 2px;
+}
+
+.first-section {
+  height: 552px;
+  width: 327px;
+  left: 24px;
+  top: 114px;
+  border-radius: 16px;
+  padding: 0;
+  margin-top: 43px;
+  margin-left: 31px;
+}
```

Comment:
- [ ] It would be nicer to add the border to the project card so your design matches the Figma design.
- Your design:
<img width="155" alt="image" src="https://user-images.githubusercontent.com/77190203/164879302-35b5eb6d-7280-4f2c-86f4-374fc95c6d2a.png">

- Figma design:
<img width="226" alt="image" src="https://user-images.githubusercontent.com/77190203/164879365-1e8f8a77-e6e5-410a-9ef3-d9cc6b0e7274.png">